### ISSUE TITLE
no display of documents

### ISSUE THREAD

**Issue by crc16** [openpaperwork/paperwork#681]
After update to version V1.2.1 on Ubuntu 16.04.3, the click of a document in the list on the left part does not change the display in the right panel. Diagnostic does not show any error.
[check.txt](https://github.com/openpaperwork/paperwork/files/1308258/check.txt)

Below is the terminal output:
 ``` INFO   paperwork.frontend.mainwindow.docs End of thumbnailing [DocThumbnailer:8]
INFO   paperwork.frontend.util.actions Action: [Open selected document]
INFO   paperwork.frontend.mainwindow.docs Showing doc 20170704_1634_53
INFO   paperwork.frontend.mainwindow.docs Document position: 0 | Number of boxes currently displayed: 50
INFO   paperwork.frontend.mainwindow  Showing document 20170704_1634_53
ERROR  paperwork.frontend.diag        === UNCATCHED EXCEPTION ===
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3759, in _show_doc_hook
    self._show_doc_internal(doc, force_refresh)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3744, in _show_doc_internal
    self._show_pages(doc)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3696, in _show_pages
    first_scan_drawer = self.__reset_page_drawers(doc)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3612, in __reset_page_drawers
    sentence=search)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/pages.py", line 1228, in __init__
    Gtk.IconLookupFlags.NO_SVG).load_icon(),
AttributeError: 'NoneType' object has no attribute 'load_icon'
ERROR  paperwork.frontend.diag        ===========================
INFO   paperwork.frontend.mainwindow.pages No box selected
INFO   paperwork.frontend.mainwindow.pages [page] found 0 boxes to highlight
INFO   paperwork.frontend.mainwindow.pages 0 boxes found
INFO   paperwork.frontend.util.actions Action: [Open selected document]
INFO   paperwork.frontend.mainwindow.docs Showing doc 20170102_1902_08
INFO   paperwork.frontend.mainwindow.docs Document position: 5 | Number of boxes currently displayed: 50
INFO   paperwork.frontend.mainwindow  Showing document 20170102_1902_08
ERROR  paperwork.frontend.diag        === UNCATCHED EXCEPTION ===
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3759, in _show_doc_hook
    self._show_doc_internal(doc, force_refresh)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3744, in _show_doc_internal
    self._show_pages(doc)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3696, in _show_pages
    first_scan_drawer = self.__reset_page_drawers(doc)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/__init__.py", line 3612, in __reset_page_drawers
    sentence=search)
  File "/usr/local/lib/python3.5/dist-packages/paperwork/frontend/mainwindow/pages.py", line 1228, in __init__
    Gtk.IconLookupFlags.NO_SVG).load_icon(),
AttributeError: 'NoneType' object has no attribute 'load_icon'
ERROR  paperwork.frontend.diag        =========================== ```

**Comment by jflesch** [comment]
Diagnostic does show the same errors than the terminal output.
It seems your system doesn't have one of the required icons: ```edit-copy```

On my system, it is located here:
```
/usr/share/icons/gnome/22x22/actions/edit-copy.png
/usr/share/icons/gnome/32x32/actions/edit-copy.png
/usr/share/icons/gnome/24x24/actions/edit-copy.png
/usr/share/icons/gnome/16x16/actions/edit-copy.png
/usr/share/icons/gnome/48x48/actions/edit-copy.png
```

**Comment by crc16** [comment]
Resolved by : apt-get install gnome-icon-theme-full
thank you for your help.

**Comment by jflesch** [comment]
Good catch. The file actually comes from ```gnome-icon-theme```.  ```paperwork-shell chkdeps``` doesn't look for it --> fixed : 1e90428e68144088ebbf634c46b320995fdbf3d1
